### ISSUE TITLE
[WebToolsE2E][Aspire] The different projects logs are displayed on the same page when click the "Logs View"

### ISSUE THREAD

**Issue by Rita003** [dotnet/aspire#4958]
**REGRESSION INFO**: Work in Aspire 8.0.2

**INSTALL STEPS**

1. Clean machine: Win11 x64 23h2 ENU
2. Install SDK 8.0.303
3. Install Aspire 8.1.0-preview.1.24367.2
4. Apply feed

**REPRO STEPS**
1. Open CMD, create an Aspire Starter App using following commands:
```
dotnet new aspire-starter -o AspireStarterApp
cd AspireStarterApp/AspireStarterApp.AppHost
```
2. Dotnet run the project in CMD, and open URL in the browser.
`dotnet run`
3. Click the apiservice project View Logs, then click the webfrontend project View Logs

**Note**
1. This issue doesn't repro when select the different project logs in the Console logs
2. This issue also repro on Aspire Starter with Redis project.

**ACTUAL**
The different projects logs are displayed on the same page
![viewlog](https://github.com/user-attachments/assets/67962bd1-6260-4574-ad19-6081bedd86a1)
![Screenshot 2024-07-18 170359](https://github.com/user-attachments/assets/e809472f-542d-41ab-a6c6-214a440ea7f4)

**EXPECTED**
The different projects logs are displayed on the different page when click the project "Logs View"

**Comment by adamint** [comment]
this should be fixed after #4957

**Comment by Rita003** [comment]
Verify this issue fixed on the latest Aspire 8.1.0-preview.1.24368.5, close it now.
